OracleSOASuite/dockerfiles/12.2.1.3/README.md

```diff
@@ -36,15 +36,15 @@ If that returns a username (which is the first field), you can skip the below us
 
 Once the `oracle` user is created, run the following commands as a root user:
 
-    # mkdir -p /u01/DockerVolume/SOAVolume/
-    # chown 1000:1000 /u01/DockerVolume/SOAVolume/
-    # chmod 700 /u01/DockerVolume/SOAVolume/
+    # mkdir -p /u01/DockerVolume/SOAVolume/SOA
+    # chown -R 1000:1000 /u01/DockerVolume/SOAVolume/
+    # chmod -R 700 /u01/DockerVolume/SOAVolume/
```

Comment:
Setting the permissions to `700` removes access to the group so there's no need to include `:1000` above. Though, I would suggest that `750` makes a little more sense (that gives write permission to the owner with read permission to the group).